sdcm/nemesis.py

```diff
@@ -3544,6 +3566,15 @@ def disrupt(self):
         self.disrupt_rolling_restart_cluster()
 
 
+class AuthenticatorSwitchAndRestart(Nemesis):
```

Comment:
Change the name to ToggleAuthenticatorType

Reply:
The nemesis only toggle between two specific types of Authenticator, not a general toggle.

Reply:
Toggle is between two only (ON/OFF).

Reply:
I guess we will only execute the new nemesis in the authorization-and-tls-ssl longevity jobs where authenticator is 'PasswordAuthenticator'.

I will change the code to record the original authenticator, and recovery at the end of nemesis.